src/modules.js

```diff
@@ -77,6 +76,10 @@ global.LibraryManager = {
       libraries.push('library_autodebug.js');
     }
 
+    if (!WASMFS) {
+      libraries.push('library_syscall.js');
```

Comment:
I actually suprised by this.. do we really only have 2 non-filesystem syscalls?  I thought we implement many more non-FS syscalls, but maybe not.

An alternative to this would would be to move all the filesystem stuff to `src/libraray_fs_syscalls.js` and keep the non-FS syscalls where they are.

But are seems to be saying that there are no such syscalls?

Might be worth renaming the file anyway of it really does contains just FS stuff?

Reply:
We have more non-filesystem system calls in `library_wasi`.

Yeah, maybe it's worth renaming `library_syscalls` to `library_fs_syscalls`... though it feels odd to me for some reason. Maybe `library_syscalls_fs`?

Reply:
After thinking on this, I'd rather not rename it. Maybe I am just very used to the old name?

Do you feel strongly @sbc100 ? And otherwise does this look good to land?